packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -239,19 +239,29 @@ export function getPropGetters<
     };
   };
 
-  const getLabelProps: GetLabelProps = (rest) => {
+  const getAutocompleteId = (instanceId: string, sourceId?: number) => {
+    return typeof sourceId !== 'undefined'
+      ? `${instanceId}-${sourceId}`
+      : instanceId;
+  };
```

Comment:
This could be extracted as an util, also not sure about the function name

Reply:
If it's used here only I think it's fine to keep it in the same file personally :D 
Simply `getId` would work for me but let's hear other opinions !

Reply:
could we consolidate between generateAutocompleteId and this function? maybe it could take more than just one argument?

Reply:
`generateAutocompleteId` is meant to be called once per autocomplete instance, prop getters are ran at every render :/
Originally they did run `generateAutocompleteId` at each render or open/close but fixed it in this commit : https://github.com/francoischalifour/autocomplete.js/commit/ce4d496d1f074d02051c1cbe1f296d2a5d6e1c1c